base-example/compose.yaml

```diff
@@ -0,0 +1,10 @@
+services:
```

Comment:
Technically we don't need this at all ... because we have ebean-test and which can start the PG docker container, set it up correctly with db/user/extensions/multiple databases/seed data, and wait for that all to complete.

Plus that works seemlessly with running tests in CI and running tests locally.

So looking ... I'm thinking the reason for adding this is that there is a configuration issue with the test setup ... I see a mismatch there between `my_app` and `my_app9` ... I reckon that is what this compose file is added.